spec/factories/apos.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 FactoryBot.define do
-  factory :apo, class: 'Cocina::Models::RequestAdminPolicy' do
+  factory :persisted_apo, class: 'Cocina::Models::RequestAdminPolicy' do
```

Comment:
Why is this `persisted_apo`? The name choice doesn't make sense to me.

Reply:
Oh wait, this is actually saved to the datastore. The "request" threw me. Can you please add comments to make the distinction between persisted and not persisted clear?